src/script/sign.h

```diff
@@ -81,4 +91,15 @@ void UpdateInput(CTxIn& input, const SignatureData& data);
  * Solvability is unrelated to whether we consider this output to be ours. */
 bool IsSolvable(const SigningProvider& provider, const CScript& script);
 
+class SignatureExtractorChecker final : public BaseSignatureChecker
```

Comment:
I don't think this class is used (or intended to be used) outside of `sign.cpp`, so perhaps move it there, inside the anonymous namespace?

Reply:
Moved